plenum/server/consensus/replica_service.py

```diff
@@ -124,3 +143,6 @@ def __repr__(self):
 
     def process_node_need_view_change(self, msg: NodeNeedViewChange):
         self._internal_bus.send(NeedViewChange(msg.view_no))
+
+    def get_time_for_3pc_batch(self):
```

Comment:
Is it here just for debug purposes?

Reply:
This is for simulation test purposes, to make time depend on TimerService only, and not on real time.